src/timecode_test_parse.rs

```diff
@@ -19,7 +19,10 @@ mod test {
         frames: i64,
         timecode: String,
         runtime: String,
-        feet_and_frames: String,
+        feet_and_frames_35mm_4perf: String,
```

Comment:
Awesome! Thanks for going through all these. It's super tedious but helps protect against breakages in the future.

I would really like these tests to assert on parsing as well as rendering. You can just add them to the `frames_sources` input `vec!` above like so:

```rust
        ParseCase{
            frames_sources: vec![
                Box::new("01:00:00:00".to_string()),
                Box::new("86400".to_string()),
                Box::new("5400+00".to_string()),
                Box::new(FeetFramesStr::new("5400+00", FilmFormat::FF35mm4perf)),
                Box::new(FeetFramesStr::new("4050+00.0", FilmFormat::FF35mm3perf)),
                Box::new(FeetFramesStr::new("2700+00", FilmFormat::FF35mm2perf)),
                Box::new(FeetFramesStr::new("4320+00", FilmFormat::FF16mm)),
                Box::new(86400i64),
                Box::new(86400u64),
                Box::new(86400i32),
                Box::new(86400u32),
                Box::new(86400isize),
                Box::new(86400usize),
            ],
```

Please make sure to test both the implicit 35mm, 4-perf input (what's currently there), and the explicit 35mm, 4perf version.

Reply:
I'm having trouble making these work for the drop-frames cases, the assertion of equal seconds is failing. Not sure exactly how to proceed with that.

Reply:
Can you copy and paste the inputs and errors you're getting?

If it's a big in the drop frame code we can merge this and I can deal with figuring out what's going on.

I *did* just fix a drop frame bug a few weeks ago. You could try rebasing on the latest dev and that might fix what you are seeing if your branch doesn't have that fix.

Reply:
The PR that fixed that is #11. Does your b branch have those changes?

Reply:
But if that doesn't fix it — and you're confident it's not a bug in your F+F code — you can skip the drop frame tests and I'll open an issue to investigate further.

Reply:
Those changes are in my dev branch, let me reapply the patch and grab an output for you.